test/functional/rpc_help.py

```diff
@@ -37,6 +43,15 @@ def run_test(self):
 
         assert_equal(titles, components)
 
+    def dump_help(self):
+        dump_dir = os.path.join(self.options.tmpdir, 'rpc_help_dump')
+        os.mkdir(dump_dir)
+        calls = [line.split(' ', 1)[0] for line in self.nodes[0].help().splitlines() if line and not line.startswith('==')]
+        for call in calls:
+            with open(os.path.join(dump_dir, call), 'w', encoding='utf-8') as f:
+                # Make sure the node can generate the help at runtime without crashing
+                f.write(self.nodes[0].help(call))
```

Comment:
Why write to file?

Reply:
There is no downside in logging the results to the temp dir. It'd make it easier to debug in case the node crashed while in the loop. Also, you could pass `--nocleanup` for manual inspection, etc...